### ISSUE TITLE
Option to disable autostart / listening for connections on boot.

### ISSUE THREAD

**Issue by mark-orion** [nitroshare/nitroshare-android#53]
The moment the android device boots up, Nitroshare listens for transfers / connections. While this might be convenient for some people, there should be an option to disable autostart for security reasons.

**Comment by benryanau** [comment]
Seconded. Not a good choice. At least a UI option?

**Comment by wheybags** [comment]
Thirded :v
Normally wouldn't add a comment for that, but since it's been two years since last interaction, I think it's worth bumping the thread.

**Comment by nathan-osman** [comment]
There is an option for this:

![Screenshot_20210206-122453](https://user-images.githubusercontent.com/1253444/107128974-bb231400-6876-11eb-9f48-f0d45c1ab79d.jpg)

Can you please confirm that this does what you described?

**Comment by wheybags** [comment]
Nope, that's not what I mean. I want to be able to receive while the app is running, but not have the app start listening on boot. ATM you can "Exit" the app from the burger menu, I want that as the default.

**Comment by nathan-osman** [comment]
Oh, I think I see what you mean — only be able to receive while the application has focus?

**Comment by wheybags** [comment]
Something along those lines yeah. I don't mind it running in the background so long as it doesn't do it automatically on boot.
